compiler/rustc_const_eval/src/const_eval/dummy_machine.rs

```diff
@@ -79,7 +80,7 @@ impl<'tcx> interpret::Machine<'tcx> for DummyMachine {
             throw_machine_stop_str!("can't access mutable globals in ConstProp");
         }
 
-        Ok(())
+        interp_ok(())
```

Comment:
Oh lol, I assumed you were just going to shadow the Result::Ok from the prelude with a free function called Ok

Reply:
Maybe I should have done that... but that could also easily have been confusing.